frontend/src/components/profile/Friend.module.css

```
@@ -0,0 +1,39 @@
+.container {
+    display: flex;
+    align-items: center;
+}
+
+.profilePicture {
+    width: var(--size-picture-profile);
+    height: var(--size-picture-profile);
+
+    border: 0.2rem solid var(--color-white);
+    filter: drop-shadow(0px -4px 16px var(--color-black));
+    border-radius: 50%;
+    transform: scale(60%);
+}
+
+.status {
+    font-size: 10px;
+    color: var(--color-black-grey);
+}
+
+.nameAndStatus {
+    display: flex;
+    flex-direction: column;
+    gap: 0.2rem;
+}
+
+.nameAndStatus h3 {
+    font-weight: 800;
+    font-size: 15px;
+    color: white;
```

We have the `--color-white` variable
> see #41 
```suggestion
    color: var(--color-white);
```

---

done : 2a51fcc166d15a60233ac96040cfc214649ead15